### DISCUSSION TITLE
Getting it running on Ubuntu

### DISCUSSION THREAD

**Discussion by lucygeneric** [fzyzcjy/flutter_convenient_test#193] [General]
Hey @fzyzcjy 
I am really excited to start using this, having a couple of issues on Ubuntu 22.04 LTS connecting to an Android Emulator

Firstly, this is not related to your package, but dart_vlc is a bit of pita https://github.com/alexmercerind/dart_vlc/issues/13.  This one is probably a direct result of this machine but I thought I would let you know in case someone else asks the Q.

The second thing is that to get the system to run I had to make this change:

`const kConvenientTestManagerHost = '10.0.2.2';`

I think thats something to do with networking on the droid emulator but I can't remember the reason behind it, you probably know more than me :D

**Comment by fzyzcjy** [comment]
> Firstly, this is not related to your package, but dart_vlc is a bit of pita https://github.com/alexmercerind/dart_vlc/issues/13. This one is probably a direct result of this machine but I thought I would let you know in case someone else asks the Q.

That is optional dependency. I use that because I want to display screen-record videos within the manager, and that is the only package that support desktop :/

> const kConvenientTestManagerHost = '10.0.2.2';

Agree. https://github.com/fzyzcjy/flutter_convenient_test/issues/194

By the way feel free to make PRs!
